### PR TITLE
Spelling scripts

### PR DESCRIPTION
**Pull request by jsoref**
This is a subset per https://github.com/zeek/zeek/pull/2499#issuecomment-1291179339

### CODE REVIEW

File: scripts/base/init-bare.zeek [hunk 1]
```diff
@@ -2541,7 +2541,7 @@ export {
 		preattr: wcc_attr_t &optional;	##< Pre operation attributes.
 		postattr: fattr_t &optional;	##< Post operation attributes.
 		size: count &optional;	##< Size.
-		commited: stable_how_t &optional;	##< TODO.
+		committed: stable_how_t &optional;	##< TODO.
```
Comment (timwoj) [inline scripts/base/init-bare.zeek:2544]: This is a functional change that should be reverted.

File: scripts/base/protocols/smb/const-nt-status.zeek [hunk 2]
```diff
@@ -1303,7 +1303,7 @@ redef SMB::statuses += {
 	[0xC0190014] = [$id="TRANSACTION_NOT_REQUESTED", $desc="The caller has called a response API, but the response is not expected because the transaction manager did not issue the corresponding request to the caller."],
 	[0xC0190015] = [$id="TRANSACTION_ALREADY_ABORTED", $desc="It is too late to perform the requested operation, because the transaction has already been aborted."],
 	[0xC0190016] = [$id="TRANSACTION_ALREADY_COMMITTED", $desc="It is too late to perform the requested operation, because the transaction has already been committed."],
-	[0xC0190017] = [$id="TRANSACTION_INVALID_MARSHALL_BUFFER", $desc="The buffer passed in to NtPushTransaction or NtPullTransaction is not in a valid format."],
+	[0xC0190017] = [$id="TRANSACTION_INVALID_MARSHALL_BUFFER", $desc="The buffer passed into NtPushTransaction or NtPullTransaction is not in a valid format."],
```
Comment (timwoj) [inline scripts/base/protocols/smb/const-nt-status.zeek:1306]: These strings actually come to us from Microsoft and should remain as-is.

File: scripts/base/protocols/smb/const-nt-status.zeek [hunk 3]
```diff
@@ -1580,7 +1580,7 @@ redef SMB::statuses += {
 	[0xC01E050C] = [$id="GRAPHICS_OPM_INVALID_HANDLE", $desc="The function failed because the caller passed in an invalid OPM user-mode handle."],
 	[0xC01E050D] = [$id="GRAPHICS_PVP_NO_MONITORS_CORRESPOND_TO_DISPLAY_DEVICE", $desc="This function failed because the GDI device passed to it did not have any monitors associated with it."],
 	[0xC01E050E] = [$id="GRAPHICS_PVP_INVALID_CERTIFICATE_LENGTH", $desc="A certificate could not be returned because the certificate buffer passed to the function was too small."],
-	[0xC01E050F] = [$id="GRAPHICS_OPM_SPANNING_MODE_ENABLED", $desc="DxgkDdiOpmCreateProtectedOutput() could not create a protected output because the video present yarget is in spanning mode."],
+	[0xC01E050F] = [$id="GRAPHICS_OPM_SPANNING_MODE_ENABLED", $desc="DxgkDdiOpmCreateProtectedOutput() could not create a protected output because the video present target is in spanning mode."],
```
Comment (timwoj) [inline scripts/base/protocols/smb/const-nt-status.zeek:1583]: These strings actually come to us from Microsoft and should remain as-is.

**Review by timwoj — CHANGES_REQUESTED** [review]
Thanks for the shorter version. I had a few comments.

**Comment by jsoref** [comment]
The only btest output I got was https://github.com/bazelbuild/bazel/issues/16413

```
% ../../auxil/btest/btest -j $(nproc)

[ 66%] scripts.base.frameworks.logging.rotate-ascii-logdir-leftover-log-rotation-shadow
[ 66%] scripts.base.frameworks.logging.rotate-ascii-logdir-leftover-log-rotation-stale-
[ 67%] scripts.base.frameworks.logging.rotate-ascii-logdir-leftover-log-rotation-stale-
[ 67%] scripts.base.frameworks.logging.rotate-default-logdir-leftover-log-rotation-shad
[ 67%] scripts.base.frameworks.logging.rotate-ascii-logdir-leftover-log-rotation-shadow
[ 67%] scripts.base.frameworks.logging.rotate-default-logdir-leftover-log-rotation-stal
[ 67%] scripts.base.frameworks.logging.rotate-default-logdir-leftover-log-rotation-stal
[ 67%] scripts.base.frameworks.logging.rotate-default-logdir-leftover-log-rotation-shad
[ 92%] scripts.policy.frameworks.files.deprecated-fields-concurrent-range-requests ... 
[ 98%] spicy.spicy-dump ... failed
1 of 1338 tests failed, 6 skipped
```

```
../../auxil/btest/btest -U spicy.spicy-dump
all 1 tests successful
```

```diff
diff --git a/testing/btest/Baseline/spicy.spicy-dump/output b/testing/btest/Baseline/spicy.spicy-dump/output
index 01a459395..01d632d28 100644
--- a/testing/btest/Baseline/spicy.spicy-dump/output
+++ b/testing/btest/Baseline/spicy.spicy-dump/output
@@ -1,4 +1,5 @@
 ### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
+ld: warning: -undefined dynamic_lookup may not work with chained fixups
 test::X {
   xs: [
     11
```

This doesn't seem like it's something desired, so I won't apply it. But, I'll drop the other items...

**Comment by jsoref** [comment]
It looks like something automagically fixed up the commits as part of its magic, not quite sure which stage did it, but it seems reasonable.

**Review by timwoj — APPROVED** [review]
This looks good to me now. Can you squash it down to a single commit for merging? The docs submodule will need updating but there's an action that will take care of that.

**Comment by jsoref** [comment]
@timwoj: done. Would you like to suggest the next little bit? This feels like roughly the right size for work...

**Comment by timwoj** [comment]
> @timwoj: done. Would you like to suggest the next little bit? This feels like roughly the right size for work...

Changes in the src directory are fine but it's going to require going through with a fine-tooth comb during the review to make sure we're not breaking any sorts of APIs in the process.

**Pull request merged**